base/v0_1/translate.go

```diff
@@ -17,75 +17,83 @@ package v0_1
 import (
 	"net/url"
 
-	"github.com/coreos/ignition/v2/config/translate"
+	"github.com/coreos/fcct/translate"
+
 	"github.com/coreos/ignition/v2/config/v3_0/types"
+	"github.com/coreos/vcontext/path"
 	"github.com/vincent-petithory/dataurl"
 )
 
-func (c Config) ToIgn3_0() (types.Config, error) {
+func (c Config) ToIgn3_0() (types.Config, translate.TranslationSet, error) {
```

Comment:
nit: The name is pretty obvious, but I'm a stickler for function documentation ... especially when it's public functions.